core/src/main/java/io/confluent/kafka/schemaregistry/storage/KafkaSchemaRegistry.java

```diff
@@ -102,6 +102,7 @@
   private ZookeeperMasterElector masterElector = null;
   private Metrics metrics;
   private Sensor masterNodeSensor;
+  private boolean zKAclsEnabled;
```

Comment:
nit: 
other zk-related variable names in general keep lowercase k (i.e. `zkBlah` rather than `zKBlah`, so maybe we should keep this consistent